test/test_helpers.go

```diff
@@ -0,0 +1,66 @@
+package test
+
+import (
+	"errors"
+	"fmt"
+	"github.com/gruntwork-io/terratest/modules/files"
```

Comment:
go import

Reply:
use command goimports to sort your imports 
https://pkg.go.dev/golang.org/x/tools/cmd/goimports

Reply:
done